### ISSUE TITLE
how do i run this 

### ISSUE THREAD

**Issue by BallisticOK** [Gunsmithy/TwitchLiveNotifier#5]
it dont want to work for me i have set up all the config to what it asked for 
but it just will not work

**Comment by schrotle** [comment]
How far did you come? Did you launch the code? What Errors occurred?

**Comment by BallisticOK** [comment]
Would not start at all and I will get you the error when I get back to my pc

**Comment by schrotle** [comment]
Have you read the Readme file? Have configured the config.ini file? Have you installed python? With this less information, I dont know what your problem is ^^. Maybe we can text on Discord. Probably faster and easier. My Discordname: Letix#4791

**Comment by Gunsmithy** [comment]
Love to see the help here guys! :)
If you figure it out, be sure to say what it was here when closing the issue so I can maybe prevent it in future.
I'm up at a cottage this weekend but feel free to reach out to me on Discord as well. Gunsmithy#1337

**Comment by BallisticOK** [comment]
Ok I added you on discord by dude just hit me up when you accept it

**Comment by Gunsmithy** [comment]
I have yet to see a Discord request so it either didn't work or you meant you added Letix.  
If you're still having issues, let me know/try again.

**Comment by schrotle** [comment]
Yes he added me. His config file wasnt configured right, But after that an import error came up. I guess he hast to reinstall python, but then he went to sleep and since then I havent heard anything from hin yet.

**Comment by Gunsmithy** [comment]
Ahhh okay.
Well I plan on migrating this to use a pyinstaller executable anyways so it's easier for people, like some of my other programs, so that should resolve that issue nicely.
I appreciate you trying to help them out while things have been hectic for me.
I'll keep this open until Atomic comments again that it's fixed or I migrate to use an executable instead.
